tests/unit/DefineCompany.spec.ts

```diff
@@ -63,7 +63,7 @@ for (const mock of mockEntityInfo) {
       expect(wrapper.find('#folio-number-header').text()).toContain('Folio / Reference Number (optional)')
     })
 
-    it(`displays company statement`, () => {
+    itIf(mock.entityType !== 'BC')(`displays company statement`, () => {
```

Comment:
Small typo.

Reply:
Which part is the typo 😆  Sorry, i'm missing it!

Reply:
> That's a method wrapper.
> Since we parameterized the tests they don’t all apply to all types, so it’s to exclude the test.